tests/clickhouse-test

```diff
@@ -360,37 +358,12 @@ def clickhouse_execute_json(
     return rows
 
 
-class Terminated(KeyboardInterrupt):
-    pass
-
-
-def signal_handler(sig, frame):
-    raise Terminated(f"Terminated with {sig} signal")
-
-
 def stop_tests():
```

Comment:
Also, maybe we should call it from signal handler? Or it will be handled gracefully and called from other parts of the code?